gitlab/resource_gitlab_project_access_token.go

```diff
@@ -0,0 +1,191 @@
+package gitlab
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectAccessToken() *schema.Resource {
+	// lintignore: XR002 // TODO: Resolve this tfproviderlint issue
```

Comment:
This resource should really have an importer since it can become deleted from the Terraform state if the PAT is not found.

Reply:
It is very complex to add an importer here, because GitLab provides the token value only when the token is created. Importing the token thus means finding a way for providing also the token value to the importer.

The same behaviour is currently implemented in

https://github.com/gitlabhq/terraform-provider-gitlab/blob/8bc5606d786d73cb9e1947d0f9bc941c2e4f61b0/gitlab/resource_gitlab_deploy_token.go#L18

I wouyld skip implementing the importer for this type of resource.

Reply:
If we did have an importer we would not set the `token` field. The purpose would be to track the resource, for example so a user could clean up the token with `terraform destroy`. But this is not a high-value use-case, so I don't mind having no importer for now.